src/main/java/com/PhotomosaicSara/image/Main.java

```diff
@@ -0,0 +1,10 @@
+package com.PhotomosaicSara.image;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        new Photomosaic(new PhotomosaicService()).process();
```

Comment:
still I don't see any recipe as I mentioned in my comment. Photomosaic and PhotomosaicService are a little bit redundant. Please work on a orchestrator solution as I provided in my comment